label_studio/data_import/api.py

```diff
@@ -380,7 +393,7 @@ class FileUploadListAPI(generics.mixins.ListModelMixin,
 
     def get_queryset(self):
         project = generics.get_object_or_404(Project.objects.for_user(self.request.user), pk=self.kwargs.get('pk', 0))
-        if project.is_draft:
+        if project.is_draft or bool_from_request(self.request.query_params, 'all', False):
```

Comment:
It's better to implement ParamSerializer here instead of bool_from_request, it will be self-documented